lib/components/quiz_gridview.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:flutter/material.dart';
+import 'package:maui/components/QuizButton.dart';
+
+class QuizGrid extends StatelessWidget {
+  final List<String> buttonData;
+
+  QuizGrid({Key key, @required this.buttonData}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    // TODO: implement build
+    return GridView.count(
+      crossAxisCount: 2,
+      crossAxisSpacing: 15.0,
+      mainAxisSpacing: 20.0,
+      shrinkWrap: true,
+      padding: EdgeInsets.all(20.0),
+      addAutomaticKeepAlives: true,
+      children: new List.generate(buttonData.length, (i) {
```

Comment:
If generating a list out of a list, prefer map function rather than generate function. Code will be cleaner